### PR TITLE
Adding shaun to member page

### PR DESCRIPTION
**Pull request by shaun3141**
Just changed the text, should be pretty low risk. Nervous to add pictures with the file path and also lazy at the moment

@regalcat  @KrashLeviathan

### CODE REVIEW

**Comment by KrashLeviathan** [comment]
Dude add a picture! Just crop and scale something to 256x256 and toss it in there. If you pull down master again, there are a couple examples from other members in there that you could use to help with the file path. Also, you'll need to shorten the description, because most of it is getting chopped off.

screen shot 2015-10-01 at 9 05 12 am

File: members/members.json [hunk 1]
```diff
@@ -31,6 +31,10 @@
     "Das Nick Cage": {
       "image": "http://www.placecage.com/g/500/500",
       "description": "Weirder than Weird Al"
+    },
+    "Shaun VanWeelden": {
+      "image": "../_images/members/shaun-vanweelden-md.jpg",
```
Comment (KrashLeviathan) [inline members/members.json]: Typo on path name. The image is shaun-weelden-md.jpg

**Comment by KrashLeviathan** [comment]
You also have merge conflicts, so you'll have to pull the club master into your branch and commit/push before your branch can be merged. Everything else looks good though.

**Comment by KrashLeviathan** [comment]
https://github.com/KrashLeviathan/club-website/blob/add_tutorials_folder_and_merge_conflicts/resources/tutorials/merge_conflicts.md

**Comment by KrashLeviathan** [comment]
@shaun3141 are you still planning on completing this PR? I know you've been pretty busy.

**Comment by frostbyte100** [comment]
Should we just close pull requests like these?

**Comment by KrashLeviathan** [comment]
Lack of activity for over three months.

**Pull request closed without merging**